BottomNavigationView/app/src/main/java/io/github/the_dagger/myapplication/MainActivity.java

```diff
@@ -0,0 +1,51 @@
+package io.github.the_dagger.myapplication;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.design.widget.BottomNavigationView;
+import android.support.v7.app.AppCompatActivity;
+import android.view.MenuItem;
+import android.widget.TextView;
+
+public class MainActivity extends AppCompatActivity {
+
+    TextView textView;
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        /**
+         * You can also implement something similar by inflating the menu item and adding it to the bottomNavigationVIew object.
+         * You can then add that object to the root of the layout.
+         */
+//        RelativeLayout relativeLayout = (RelativeLayout) findViewById(R.id.activity_main);
+//        BottomNavigationView bottomNavigationView = new BottomNavigationView(this);
+//        bottomNavigationView.inflateMenu(R.menu.my_navigation_items);
+//        relativeLayout.addView(bottomNavigationView);
+        textView = (TextView) findViewById(R.id.text_view);
```

Comment:
show a working example of this in a new activity with this info in textview and a button to link to the new activity.